root/edit/details/merge_releases.tt

```diff
@@ -2,8 +2,8 @@
   <tr>
     <td>[% link_entity(release) %]</td>
     <td>[% artist_credit(release.artist_credit) %]</td>
-    <td>[% html_escape(release.combined_format_name) %]</td>
-    <td>[% release.combined_track_count %]</td>
+    <td>[% html_escape(release.combined_format_name) || l('[missing media]') %]</td>
+    <td>[% release.combined_track_count || l('-') %]</td>
```

Comment:
In #1242 I used `lp('-', 'missing data')` in some places instead. Maybe we can do the same here and below?